add protocols to main page

FireFart opened the pull request:
This PR adds TLS1.0, TLS1.1, SSLv2 and SSLv3 to the front page under the client section. Also the Dashboard now checks SSLv3 and SSLv2

FireFart commented:
I signed it!

On 2017-06-19 12:16, googlebot wrote:
> Thanks for your pull request. It looks like this may be your first
> contribution to a Google open source project. Before we can look at your
> pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 *Please visit https://cla.developers.google.com/ to sign.*
> 
> Once you've signed, please reply here (e.g. |I signed it!|) and we'll
> verify. Thanks.
> 
> ------------------------------------------------------------------------
> 
>   * If you've already signed a CLA, it's possible we don't have your
>     GitHub username or you're using a different email address. Check
>     your existing CLA data <https://cla.developers.google.com/clas> and
>     verify that your email is set on your git commits
>     <https://help.github.com/articles/setting-your-email-in-git/>.
>   * If your company signed a CLA, they designated a Point of Contact who
>     decides which employees are authorized to participate. You may need
>     to contact the Point of Contact for your company and ask to be added
>     to the group of authorized contributors. If you don't know who your
>     Point of Contact is, direct the project maintainer to
>     go/cla#troubleshoot.
>   * In order to pass this check, please resolve this problem and have
>     the pull request author add another comment and the bot will run again.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/chromium/badssl.com/pull/296#issuecomment-309398508>, or
> mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGbQQqOoUyB3bhB3POEa7R26BQG_Yntks5sFkpigaJpZM4N-BkI>.
>

lgarron commented on domains/misc/badssl.com/index.html:
```diff
@@ -81,6 +81,13 @@ <h2 id="miscellaneous"><span class="emoji">✏️</span>HTTP Input</h2>
     <a href="http://http-credit-card.{{ site.domain }}/" class="bad"><span class="icon"></span>http-credit-card</a>
   </div>
   <div class="group">
+    <h2 id="protocol"><span class="emoji">↔️</span>Protocol</h2>
```
Could you put this after the `Key Exchange` section to balance the 2-column layout?

(I like the emoji!)

lgarron commented on domains/misc/badssl.com/index.html:
```diff
@@ -81,6 +81,13 @@ <h2 id="miscellaneous"><span class="emoji">✏️</span>HTTP Input</h2>
+    <h2 id="protocol"><span class="emoji">↔️</span>Protocol</h2>
+    <a href="https://ssl-v2.{{ site.domain }}/" class="bad"><span class="icon"></span>SSLv2</a>
+    <a href="https://ssl-v3.{{ site.domain }}/" class="bad"><span class="icon"></span>SSLv3</a>
+    <a href="https://tls-v1-0.{{ site.domain }}/" class="dubious"><span class="icon"></span>TLS 1.0</a>
```
Nits:
- Link to the port directly.
- Use the subdomain label (`tls-v1-0`) as the link text

lgarron commented on domains/misc/badssl.com/index.html:
```diff
@@ -81,6 +81,13 @@ <h2 id="miscellaneous"><span class="emoji">✏️</span>HTTP Input</h2>
+    <a href="https://ssl-v2.{{ site.domain }}/" class="bad"><span class="icon"></span>SSLv2</a>
+    <a href="https://ssl-v3.{{ site.domain }}/" class="bad"><span class="icon"></span>SSLv3</a>
+    <a href="https://tls-v1-0.{{ site.domain }}/" class="dubious"><span class="icon"></span>TLS 1.0</a>
+    <a href="https://tls-v1-1.{{ site.domain }}/" class="dubious"><span class="icon"></span>TLS 1.1</a>
```
Same nits.

lgarron commented on domains/misc/badssl.com/index.html:
```diff
@@ -81,6 +81,13 @@ <h2 id="miscellaneous"><span class="emoji">✏️</span>HTTP Input</h2>
   </div>
   <div class="group">
+    <h2 id="protocol"><span class="emoji">↔️</span>Protocol</h2>
+    <a href="https://ssl-v2.{{ site.domain }}/" class="bad"><span class="icon"></span>SSLv2</a>
```
Remove SSLv2 and SSLv3 for now.

lgarron commented on domains/misc/badssl.com/dashboard/sets.js:
```diff
@@ -19,6 +19,8 @@ var sets = [
       {subdomain: "superfish"},
       {subdomain: "edellroot"},
       {subdomain: "dsdtestprovider"},
+      {subdomain: "ssl-v2", port: 1002},
```
Remove SSLv2 and SSLv3 for now.

lgarron commented:
Thanks for the PR!

Unfortunately, our `ssl-v2` and `ssl-v3` implementations aren't working, and disabled for now. Could you take those, or help with #20 to get them working?

FireFart commented:
@lgarron I added the suggested changes. I also fixed a copy paste error on the `certificate-transparency` id.

I will have a look at #20 later to see if I can come up with a solution for this

FireFart commented:
@lgarron any chance to get this merged?

lgarron commented:
Yep, looks good!

FireFart commented:
awesome thanks!
